protected/humhub/modules/user/behaviors/Followable.php

```diff
@@ -43,7 +43,7 @@ class Followable extends Behavior
     public function getFollowRecord($userId)
     {
         $userId = ($userId instanceof User) ? $userId->id : $userId;
-        return Yii::$app->runtimeCache->getOrSet(__METHOD__ . $userId, function () use ($userId) {
+        return Yii::$app->runtimeCache->getOrSet(__METHOD__ . $this->owner->id, function () use ($userId) {
```

Comment:
@sebastianSchmidt86 I see the record `Follow` is found by two values `$this->owner->getPrimaryKey()` and `$userId`, so I think it should be cached by these 2 values like this:

- `__METHOD__ . $this->owner->getPrimaryKey() . '-' . $userId`